subtests/docker_cli/create/create_tmpfs.py

```diff
@@ -0,0 +1,38 @@
+"""
+Create container with --tmpfs option and inspect it
+"""
+
+from create import create_base
+from dockertest.output import DockerVersion
+from dockertest.dockercmd import DockerCmd
+from dockertest.output import mustpass
+
+
+class create_tmpfs(create_base):
+
+    def initialize(self):
+        super(create_tmpfs, self).initialize()
+        # Minimum docker version 1.10 is required for --tmpfs
+        DockerVersion().require_server("1.10")
+
+    def run_once(self):
+        super(create_tmpfs, self).run_once()
+
+    def postprocess(self):
+        # Inspect container for --tmpfs
```

Comment:
A good comment explains "why" you are doing what follows, not "what".  We can tell "what" just by reading the code :smile: